modules/RTC/ScreenObtainer.js

```diff
@@ -262,12 +260,12 @@ const ScreenObtainer = {
 
         if (desktopSharingFirefoxMaxVersionExtRequired === -1
             || (desktopSharingFirefoxMaxVersionExtRequired >= 0
-                && RTCBrowserType.getFirefoxVersion()
-                    <= desktopSharingFirefoxMaxVersionExtRequired)) {
+                && !browser.isVersionGreaterThan(
+                    desktopSharingFirefoxMaxVersionExtRequired))) {
```

Comment:
To confirm, it's okay that this is checking greater than only and not greater than or equal too?

Reply:
It is checking <= because it is "not grater than". You probably missed `!`.